### PR TITLE
Add WebAssembly exception handling with Safari data

### PR DESCRIPTION
**Pull request by jdatapple**
#### Summary
Adds builtins for `WebAssembly.Exception` and `WebAssembly.Tag` for WASM exception handling.

#### Test results and supporting details
See [Safari 15.2 Release Notes](https://developer.apple.com/documentation/safari-release-notes/safari-15_2-release-notes)

### CODE REVIEW

**Comment by Elchi3** [comment]
I requested this to be a known spec in browser-specs in https://github.com/w3c/browser-specs/issues/508

**Comment by queengooborg** [comment]
The spec URLs have landed in a browser-specs release, so this PR is ready to go after a quick merge of the latest `main` branch!

File: javascript/builtins/webassembly/Exception.json [hunk 1]
```diff
@@ -0,0 +1,229 @@
...
+              "experimental": true,
+              "standard_track": true,
+              "deprecated": false
+            }
+          },
+          "Exception": {
+            "__compat": {
+              "description": "<code>Exception()</code> constructor",
+              "mdn_url": "https://developer.mozilla.org/docs/Web/JavaScript/Reference/Global_Objects/WebAssembly/Exception/Exception",
+              "spec_url": "https://webassembly.github.io/exception-handling/js-api/index.html#runtime-exceptions",
```
Comment (Elchi3) [inline javascript/builtins/webassembly/Exception.json:63]:
```suggestion
              "spec_url": "https://webassembly.github.io/exception-handling/js-api/index.html#dom-exception-exception",
```

File: javascript/builtins/webassembly/Exception.json [hunk 2]
```diff
@@ -0,0 +1,229 @@
...
+                "experimental": true,
+                "standard_track": true,
+                "deprecated": false
+              }
+            }
+          },
+          "getArg": {
+            "__compat": {
+              "mdn_url": "https://developer.mozilla.org/docs/Web/JavaScript/Reference/Global_Objects/WebAssembly/Exception/getArg",
+              "spec_url": "https://webassembly.github.io/exception-handling/js-api/index.html#runtime-exceptions",
```
Comment (Elchi3) [inline javascript/builtins/webassembly/Exception.json:118]:
```suggestion
              "spec_url": "https://webassembly.github.io/exception-handling/js-api/index.html#dom-exception-getarg",
```

File: javascript/builtins/webassembly/Exception.json [hunk 3]
```diff
@@ -0,0 +1,229 @@
...
+                "experimental": false,
+                "standard_track": true,
+                "deprecated": false
+              }
+            }
+          },
+          "is": {
+            "__compat": {
+              "mdn_url": "https://developer.mozilla.org/docs/Web/JavaScript/Reference/Global_Objects/WebAssembly/Exception/is",
+              "spec_url": "https://webassembly.github.io/exception-handling/js-api/index.html#runtime-exceptions",
```
Comment (Elchi3) [inline javascript/builtins/webassembly/Exception.json:173]:
```suggestion
              "spec_url": "https://webassembly.github.io/exception-handling/js-api/index.html#dom-exception-is",
```

File: javascript/builtins/webassembly/Tag.json [hunk 4]
```diff
@@ -0,0 +1,174 @@
...
+              "experimental": true,
+              "standard_track": true,
+              "deprecated": false
+            }
+          },
+          "Tag": {
+            "__compat": {
+              "description": "<code>Tag()</code> constructor",
+              "mdn_url": "https://developer.mozilla.org/docs/Web/JavaScript/Reference/Global_Objects/WebAssembly/Tag/Tag",
+              "spec_url": "https://webassembly.github.io/exception-handling/js-api/index.html#tags",
```
Comment (Elchi3) [inline javascript/builtins/webassembly/Tag.json:63]:
```suggestion
              "spec_url": "https://webassembly.github.io/exception-handling/js-api/index.html#dom-tag-tag",
```

File: javascript/builtins/webassembly/Tag.json [hunk 5]
```diff
@@ -0,0 +1,174 @@
...
+                "experimental": true,
+                "standard_track": true,
+                "deprecated": false
+              }
+            }
+          },
+          "type": {
+            "__compat": {
+              "mdn_url": "https://developer.mozilla.org/docs/Web/JavaScript/Reference/Global_Objects/WebAssembly/Tag/type",
+              "spec_url": "https://webassembly.github.io/exception-handling/js-api/index.html#tags",
```
Comment (Elchi3) [inline javascript/builtins/webassembly/Tag.json:118]:
```suggestion
              "spec_url": "https://webassembly.github.io/exception-handling/js-api/index.html#dom-tag-type",
```

**Review by Elchi3 — APPROVED** [review]

**Pull request merged**